src/styles.scss

```diff
@@ -1 +1,10 @@
 /* You can add global styles to this file, and also import other style files */
+
+@import '~@angular/material/prebuilt-themes/deeppurple-amber.css';
```

Comment:
Default material theme style (3 others to chose from), we can create our own, change this, or whatever in the future.

https://material.angular.io/guide/theming